examples/sql-database-projects/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.AppHost/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.AppHost.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Sdk Name="Aspire.AppHost.Sdk" Version="$(AspireVersion)" />
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireHost>true</IsAspireHost>
+    <UserSecretsId>e290de9f-eab3-428b-bd01-f53b072f8406</UserSecretsId>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Aspire.Hosting.AppHost" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\src\CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects\CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.csproj" IsAspireProjectResource="False" />
+    <ProjectReference Include="..\SdkProject\SdkProject.csproj" />
```

Comment:
I just noticed in the build output this warning:

> Warning: /home/runner/.nuget/packages/aspire.hosting.apphost/9.0.0-rc.1.24511.1/build/Aspire.Hosting.AppHost.targets(178,5): warning ASPIRE004: '../SdkProject/SdkProject.csproj' is referenced by an Aspire Host project, but it is not an executable. Did you mean to set IsAspireProjectResource="false"? [/home/runner/work/Aspire/Aspire/examples/sql-database-projects/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.AppHost/CommunityToolkit.Aspire.Hosting.SqlDatabaseProjects.AppHost.csproj]

The problem here is that Aspire does validation of the projects that are referenced and if they are not executable projects, then it expects the `IsAspireProjectResource` to be set to `false`.

Here's where the warning comes from: https://github.com/dotnet/aspire/blob/main/src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.targets#L178-L180

My _assumption_ is that Aspire doesn't want to generate any classes under `Projects` (like `Project.MyWebApp`) for anything that it isn't able to "start", which are executable projects, so it raises the error.

Given that it does make sense for us to be using the generated `Project` here for a clean usage on the `AddSqlProject` method I want to bring this up to the Aspire team to get their input.

Reply:
This is also noted in the readme

Reply:
I've had a chat with one of the Aspire folks today and yeah, kind of no "solution" for it at present. I'll give it another 24 hours (since I posted it after hours US) but we'll have to accept that this warning will exist for the time being I believe.

Unless someone comes up with some crazy solution by tomorrow I'll be merging this PR as the integration looks slick.